### ISSUE TITLE
Force installing an old modversion for another KSP version makes CKAN think it's AD

### ISSUE THREAD

**Issue by Dazpoet** [KSP-CKAN/CKAN#937]
So as is common knowledge one can do `ckan install <identifier>=<version>` to install a certain version of a mod. Being a person of the let's-break-things mentality I did a `ckan install NearFuturePropulsion=0.4.0` on a KSP 1.0.2 install just to see what happens. CKAN happily installed NFP and CrossFeedEnabler (which is a dependency of NFP).

Then things got weird, NFP got marked as AD in CKAN as if I had manually installed it rather than having CKAN handle the install. What's even weirder though is that CKAN also marked CrossFeedEnabler as AD even though it is available for KSP 1.0.2 and is readily available for installation.

Problem found in CKAN 1.6.16 running Windows 7 and KSP 1.0.2.

Reproduction steps:

1) Have KSP 1.0.2 and use CKAN-Commandline client
2) run `ckan install NearFuturePropulsion=0.4.0`
3) run `ckan list` and notice how both NFP and CrossFeedEnabler are marked AD
4) check GUI and notice that the same issue persists there.

**Comment by pjf** [comment]
This is *super weird*. I haven't tried to reproduce this yet, but if we can it's definitely a bug, and one of the stranger ones at that.

**Comment by Dazpoet** [comment]
Unable to reproduce on first try with 1.6.17.

**Comment by Dazpoet** [comment]
Unable to reproduce on a fresh testinstall aswell (with both 1.6.16 and 1.6.17), I did have this reproducing fine in one of my testinstalls when I wrote the reproduction steps but alas that one has been purged since then. I think it might have had an AD ModuleManager in it at the time but even with one of those I've been unable to reproduce anything near the behaviour in the OP.

I'm closing this due to inability to reproduce even on the machine that first had the problem.
